### DISCUSSION TITLE
A platform for people to collectively decide on strategies and goals

### DISCUSSION THREAD

**Discussion by bencyrus** [WeAreMahsaAmini/Core#69] [Ideas]
We can build a platform so people can say what it is they wanna do, and the most popular of them get amplified so we can get a direction and a shared vision. What a leader does is conveying a vision. If we don't want a leader, there needs to be another way to get everyone on the same boat and towards the same goal.
Although I understand such a platform could be influenced by bots, so we can have verifications like each person needs to attach a few other social media they have, and they can get invites and basically get people to vouch for them. Sort of like Clubhouse's beginning strategy but more strict. or we can onboard users like that but weigh their votes based on the connections they have and the social media they have added.

This is a raw idea, but we are using social media for all our efforts, and wonderful ideas have come up so far. We can use that as a form of forced democracy and a leading tool as well.

**Comment by mrnobud** [comment]
We can build a DAO, that would easily limit/eliminate bots and ensure authenticity 
We can build it on a cheap blockchain, or we can build a token and anyone with that token/NFT can get access to the DAO with voting, etc. 

But the problem with these ideas (even a web2 solution like the one you are suggesting is access to technology). IF we want people to truly participate and vote or contribute, we need them to have access to internet.

**Reply by bencyrus** [reply]
I am not very familiar with blockchain technology exactly but we need real Iranian people there or at least an algorithm that voices their opinions or votes based on their popularity/followers. Not every user can get the same weight in votes or the polls and directions can be easily highjacked and abused. 

-a vouching mechanism
-proper weighting based on the fanbase

are keys IMO

**Comment by kaveh6** [comment]
How is it different than twitter ?? People are organising through twitter spaces and clubhouse .

**Reply by bencyrus** [reply]
It is not to pull them over to our platform but more like a voting system for defining the goals and visions collectively. Each person is now sharing what they want on multiple social media. Few people are giving directions like fariborzkaramizand but not everyone gets the messages and not everyone supports them.

just an example of what I am thinking about:
Imagine a fariborz karamizand says it is a good idea to attack the police stations at this point from different directions. And I say it is better to occupy farmandari's instead. Some say let's go to the parliament. Normally people won’t know what to do and all the energy is wasted. But now in such a platform, everyone voices and categorizes their action plans and people vote for those plans and the most voted ones naturally get picked. More importantly in that way more influential people (let's say Ali Karimi) will know what to encourage people do to.

a lot like our own idea lists here. The most voted ones get executed.

**Comment by ziXet** [comment]
Guys, internet is completely blocked in Iran! Please focus on practical ideas.

**Reply by bencyrus** [reply]
These two issues are unrelated. Even with a form of connection if people are not on the same page and don’t have an action plan, nothing can be done, and efforts will be wasted.

I am not a network expert. I am trying out different ways suggested, but nothing gets done even with the internet if everyone is not aiming for the same thing.
